EventsManagementService/EventManagementService.Domain/Models/JobEvent.cs

```diff
@@ -76,9 +78,14 @@ public List<string> IsValidEventToUpdate()
                 invalidErr.Add($"Invalid PetServiceId: {PetServiceId}.");
             }
 
-            if (EventDate == null)
+            if (EventStartTime == null)
+            {
+                invalidErr.Add("Please set a start date and time for event.");
+            }
+
+            if (EventEndTime == null)

```

Comment:
Not sure you need this.

Think about it, even if clients update start time, they will not pass in end time. The backend will be the one that is recomputing it.